doc/source/wsrep-system-index.rst

```diff
@@ -294,265 +436,454 @@ This variable is used to set the `notification command <http://galeracluster.com
+   (or disable it completely
+   if you don't mind joiner node waiting for state transfer indefinitely).
 
 .. variable:: wsrep_sst_donor_rejects_queries
 
-   :cli: Yes
+   :cli: ``--wsrep-sst-donor-rejects-queries``
    :conf: Yes
    :scope: Global
    :dyn: Yes
-   :default: OFF
+   :default: ``OFF``
+
+Defines whether the node should reject blocking client sessions
+when it is serving as a donor during a blocking state transfer method
+(when :variable:`wsrep_sst_method` is set to ``mysqldump`` or ``rsync``).
+This is disabled by default, meaning that the node accepts such queries.
 
-This variable can be used to reject blocking client sessions on a node
-serving as a donor during a blocking state transfer method.
-Queries will return the ``UNKNOWN COMMAND`` error code.
+If you enable this variable, queries will return the ``Unknown command`` error.
 This can be used to signal load-balancer that the node isn't available.
 
 .. variable:: wsrep_sst_method
 
-   :cli: Yes
+   :cli: ``--wsrep-sst-method``
    :conf: Yes
    :scope: Global
    :dyn: Yes
-   :default: xtrabackup-v2
-   :recommended: xtrabackup-v2
+   :default: ``xtrabackup-v2``
 
-This variable sets up the method for taking the State Snapshot Transfer (SST). Available values are:
+Defines the method or script for :ref:`state_snapshot_transfer` (SST).
 
-* ``xtrabackup``: Uses Percona XtraBackup to perform the SST, this method requires :variable:`wsrep_sst_auth` to be set up with <user>:<password> which |XtraBackup| will use on donor. Privileges and permissions needed for running |XtraBackup| can be found `here <http://www.percona.com/doc/percona-xtrabackup/innobackupex/privileges.html#permissions-and-privileges-needed>`_.
+Available values are:
 
-* ``xtrabackup-v2``: This is the same as ``xtrabackup``, except that it uses newer protocol, hence is not compatible. This is the **recommended** option for PXC 5.5.34 and above. For more details, please check :ref:`xtrabackup_sst` and :ref:`errata`. This is also the default SST method. For SST with older nodes (< 5.5.34), use ``xtrabackup`` as the SST method.
+* ``xtrabackup-v2``: Uses |Percona XtraBackup| to perform SST.
+  This method requires :variable:`wsrep_sst_auth`
+  to be set up with credentials (``<user>:<password>``) on the donor node.
+  Privileges and perimssions for running |Percona XtraBackup|
+  can be found `in Percona XtraBackup documentation
+  <https://www.percona.com/doc/percona-xtrabackup/2.4/using_xtrabackup/privileges.html>`_.
 
-  .. note:: This method is currently recommended if you have ``innodb-log-group_home-dir/innodb-data-home-dir`` in your config. Refer to :option:`sst-special-dirs` for more information.
+  This is the **recommended** and default method for |PXC|.
+  For more information, see :ref:`xtrabackup_sst`.
 
-* ``rsync``: Uses ``rsync`` to perform the SST, this method doesn't use the :variable:`wsrep_sst_auth`
+  .. note:: If you have ``innodb_data_home_dir`` and ``innodb_log_home_dir``
```

Comment:
The sst-special-dirs option was removed (for 5.6 and 5.7), so this section can be removed.